site/en/guides/index.md

```diff
@@ -1,3 +1,9 @@
+---
+layour: scrolling
```

Comment:
Remove this, the guides page doesn't need it, only the codelabs do.

For this page, from the internal bug, you need to update this content (at the bottom of the page):

"Learn how we test OpenThread and what user testing tools are available on our
[Testing](https://openthread.io/testing/) page, and browse the latest OpenThread peformance quality
metrics on our [Quality Dashboards](https://openthread.io/testing/quality-dashboard)."

To 

"Learn how we test OpenThread and what user testing tools are available on our
[Testing](https://openthread.io/testing/) page."

Reply:
ok, plus it wouldn't have worked anyway since i made a typo on that file.  -)

Reply:
Hi Jeff - I have made those two changes now in the guides file,